letsencrypt/plugins/manual.py

```diff
@@ -130,7 +130,7 @@ def _perform_single(self, achall):
         if self.conf("test-mode"):
             logger.debug("Test mode. Executing the manual command: %s", command)
             # sh shipped with OS X does't support echo -n
-            if sys.platform == "darwin":
+            if sys.platform == "darwin":   # pragma: no cover
                 executable = "/bin/bash"
             else:
                 executable = None
```

Comment:
It is cheating, indeed, but comment about typo is still true: Darwin branch could say `exceutable = ...` and we could not notice the typo with this setup. Mocking `sys.platform` is ideal - do you have time to implement it? :)